okex-price/src/order_book/book.rs

```diff
@@ -204,8 +211,8 @@ impl OrderBookCache {
         Ok(())
     }
 
-    pub fn latest(&self) -> CompleteOrderBook {
-        self.current.clone()
+    pub fn latest(&self) -> &CompleteOrderBook {
+        self.current.as_ref()
```

Comment:
I'm really surprised. Doesn't `&self.current` work?

Reply:
I am uncertain why it didn't work before but I updated my IDE and this works now. Also removed the implementation of `AsRef`